src/Teams/Models/TeamMember.cs

```diff
@@ -5,18 +5,12 @@ namespace Teams.Models
 {
     public class TeamMember
     {
-        [Column("Id", TypeName = "int")]
-        [Required(AllowEmptyStrings = false)]
-        [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+        [Column("Id", TypeName = "int"), Key, Required, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
-        [Column("TeamId", TypeName = "int")]
-        [Required(AllowEmptyStrings = false)]
-        [ForeignKey("Team_Id")]
+        [Column("TeamId", TypeName = "int"), ForeignKey(nameof(Team)), Required] 
+        public int TeamId { get; set; }
         public virtual Team Team { get; set; }
-        [Column("MemberId", TypeName = "nvarchar(450)")]
-        [ForeignKey("AspNetUsers_Id")]
-        [Required(AllowEmptyStrings = false)]
+        [Column("MemberId", TypeName = "nvarchar(450)"), ForeignKey("AspNetUsers"), Required]
```

Comment:
Where is the virtual property of IdentityUser? Again this were discussed many times and I showed all required models on our last meeting